packages/twenty-server/src/modules/calendar/repositories/calendar-event-attendee.repository.ts

```diff
@@ -90,20 +90,70 @@ export class CalendarEventAttendeeRepository {
     const dataSourceSchema =
       this.workspaceDataSourceService.getSchemaName(workspaceId);
 
+    const handles = calendarEventAttendees.map(
```

Comment:
imho there is too much logic for a repository here, it should live in a service. Also you are querying "person" inside calendar-event-attendee which kind of break our pattern